projects/igniteui-angular/ng-package.json

```diff
@@ -12,6 +12,7 @@
     "igniteui-trial-watermark",
     "lodash-es",
     "uuid",
-    "@igniteui/material-icons-extended"
+    "@igniteui/material-icons-extended",
+    "igniteui-theming"
```

Comment:
You also need to update the `ng-package.prod.json` and the `package.json` at the same level :)